src/applications/financial-status-report/components/ReviewCardField.jsx

```diff
@@ -43,7 +44,7 @@ const ReviewCardField = ({
 
   useEffect(() => {
     // Throw an error if there’s no viewComponent (should be React component)
-    if (typeof get('ui:options.viewComponent', uiSchema) !== 'function') {
+    if (!isReactComponent(get('ui:options.viewComponent', uiSchema))) {
```

Comment:
getting a prop type error on this one may need to update `PropTypes.func` -> `PropTypes.elementType` for the viewComponent [here](https://github.com/department-of-veterans-affairs/vets-website/blob/1b9605945cd90eac51170c8af6010cba9d1d0768/src/applications/financial-status-report/components/ReviewCardField.jsx#L228)


Warning: Failed prop type: Invalid prop `uiSchema.ui:options.viewComponent` supplied to `ReviewCardField`.

Reply:
Oh, good catch, I haven't touched any prop types, I will make a pass through for those.